components/account/UserButtons.vue

```diff
@@ -82,7 +82,11 @@
         this.$bvModal.show(this.addItemToSetModalId);
       },
       async toggleLiked() {
-        await (this.liked ? this.unlike() : this.like());
+        if (this.$auth.loggedIn) {
+          await (this.liked ? this.unlike() : this.like());
+        } else {
+          this.$auth.loginWith('keycloak');
+        }
```

Comment:
I don't think this change is needed.
Attempting to like an item while not logged in will trigger the authentication in the store.
Revert this and the changes to the specs, unless there's a case I'm missing?

Reply:
it was showing me an error when I was logged out, but now I'm unable to replicate. Reverting